README.md

```diff
@@ -108,7 +108,7 @@ in the [mdn/content repository](https://github.com/mdn/content).
 
 Yari does a number of things, the most important of which is to render and serve
 the MDN content found in the [content repo](https://github.com/mdn/content).
-Each document is stored as an `index.html` file that contains metadata presented
+Each document is stored as a `index.md` or `index.html` file that contains metadata presented
```

Comment:
Unfortunately, `markdown-lint` doesn't like this long line.

Could you rewrap this paragraph, starting from this line?